src/gravel/controllers/resources/inventory.py

```diff
@@ -37,10 +50,26 @@ async def probe(self) -> None:
         diff: int = int(time.monotonic()) - start
         logger.info(f"=> inventory probing took {diff} seconds")
         self._latest = nodeinfo
+        await self._publish()
 
-    async def latest(self) -> Optional[NodeInfoModel]:
+    @property
+    def latest(self) -> Optional[NodeInfoModel]:
```

Comment:
This will fix my comment on the previous commit. (Sorry just going through this in commit order).